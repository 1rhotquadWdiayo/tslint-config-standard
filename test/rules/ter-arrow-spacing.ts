function exec (cb: () => void) {
  return cb()
}

exec( ()  => undefined)
