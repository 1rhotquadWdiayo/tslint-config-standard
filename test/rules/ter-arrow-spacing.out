
ter-arrow-spacing.ts[5, 11]: Multiple spaces found before '=>'.
